src/ipc-handlers.ts

```diff
@@ -459,6 +471,17 @@ export async function clearAuthenticationToken() {
 	return oauthClient.clearAuthenticationToken();
 }
 
+export async function exportSite(
+	_event: IpcMainInvokeEvent,
+	options: ExportOptions
+): Promise< void > {
+	try {
+		await exportBackup( options );
+	} catch ( e ) {
+		Sentry.captureException( e );
```

Comment:
Following [this comment](https://github.com/Automattic/studio/pull/365#discussion_r1686157322), it might be useful to throw the original error to let the UI handle it. However, we can do it in the PR that implements the UI in favor of merging this PR sooner.